k-distribution/tests/regression-new/map-symbolic-tests-haskell/inkeys-6-spec.k.out

```diff
@@ -0,0 +1,16 @@
+  <k>
+    inkeysResult ( Y:MyId in_keys ( M
+    ( X:MyId |-> 0 ) ) ) ~> .
```

Comment:
This is the expected output on the Java backend as well, but I don't understand what this really tests.